mm2src/coins/lp_coins.rs

```diff
@@ -294,6 +295,8 @@ pub enum RawTransactionError {
     Transport(String),
     #[display(fmt = "Hash does not exist: {}", _0)]
     HashNotExist(String),
+    #[cfg(target_arch = "wasm32")]
+    MetamaskError(MetamaskRpcError),
```

Comment:
Yes, it can be mapped to `Transport`. The reason is that the `MetamaskRpcError` may contain a useful information for the GUI. But I agree with you, this will make the code better

Reply:
Done